package.json

```diff
@@ -25,35 +28,31 @@
     "raven-js": "^3.20.1",
     "react": "^16.1.1",
     "react-dom": "^16.1.1",
+    "react-router-dom": "^4.2.2",
     "webrtc-adapter": "^6.0.2"
   },
-  "resolutions": {
-    "webpack-sources": "1.0.1"
-  },
   "devDependencies": {
     "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^7.1.3",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "chokidar": "^1.7.0",
-    "crc": "^3.5.0",
-    "cross-env": "^5.1.1",
-    "css-loader": "^0.28.7",
+    "cross-env": "^5.1.3",
+    "css-loader": "^0.28.10",
+    "dotenv": "^5.0.1",
     "eslint": "^4.10.0",
     "eslint-config-prettier": "^2.6.0",
     "eslint-plugin-prettier": "^2.3.1",
-    "file-loader": "^1.1.5",
-    "filemanager-webpack-plugin": "^1.0.16",
-    "fs-extra": "^4.0.2",
-    "glob": "^7.1.2",
-    "handlebars": "^4.0.11",
-    "optimize-css-assets-webpack-plugin": "^3.2.0",
+    "extract-text-webpack-plugin": "4.0.0-alpha.0",
+    "file-loader": "^1.1.10",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "webpack-contrib/html-webpack-plugin",
+    "lodash": "^4.17.5",
     "prettier": "^1.7.0",
-    "style-loader": "^0.19.0",
-    "uglifyjs-webpack-plugin": "^1.1.2",
-    "webpack": "^3.6.0",
-    "webpack-dev-server": "^2.9.3",
-    "webpack-merge": "^4.1.0",
-    "write-file-webpack-plugin": "^4.2.0"
+    "rimraf": "^2.6.2",
```

Comment:
sigh

Reply:
Haha, what's crazy is how large this package is just to get proper support on Windows: https://github.com/isaacs/rimraf/blob/master/rimraf.js